config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
+OnCalendar=daily
+# Start immediately if last start was missed (ie when machine was down)
+Persistent=true
+# Delay timer randomly to spread out load (centos-7.3 and up only)
+RandomizedDelaySec=30min
+# Do not coalesce with other timers, no need to save power during a backup
+AccuracySec=1us
```

Comment:
I don't see any benefit in this one, as the timer is randomized over 30 minutes anyway and for a backup job there's no need to start it within a micro second accuracy window. I would leave it at the default of [1 minute](https://www.freedesktop.org/software/systemd/man/systemd.timer.html#AccuracySec=).